src/main/resources/fxml_files/AddTransactionPopup.fxml

```diff
@@ -1,154 +1,141 @@
             </font>
         </Text>
-        <Text fx:id="accountLabel" strokeType="OUTSIDE" strokeWidth="0.0" text="Account:" textAlignment="RIGHT"
-              wrappingWidth="120.6640625" GridPane.rowIndex="3">
+        <Text fx:id="accountLabel" strokeType="OUTSIDE" strokeWidth="0.0" text="Account:" textAlignment="RIGHT" wrappingWidth="120.6640625" GridPane.rowIndex="3">
             <font>
-                <Font name="Calibri" size="24.0"/>
+                <Font name="Calibri" size="24.0" />
             </font>
         </Text>
-        <Text fx:id="categoryLabel" strokeType="OUTSIDE" strokeWidth="0.0" text="Category:" textAlignment="RIGHT"
-              wrappingWidth="120.6640625" GridPane.rowIndex="4">
+        <Text fx:id="categoryLabel" strokeType="OUTSIDE" strokeWidth="0.0" text="Category:" textAlignment="RIGHT" wrappingWidth="120.6640625" GridPane.rowIndex="4">
             <font>
-                <Font name="Calibri" size="24.0"/>
+                <Font name="Calibri" size="24.0" />
             </font>
         </Text>
-        <Text fx:id="notesLabel" strokeType="OUTSIDE" strokeWidth="0.0" text="Notes:" textAlignment="RIGHT"
-              wrappingWidth="120.6640625" GridPane.rowIndex="5">
+        <Text fx:id="notesLabel" strokeType="OUTSIDE" strokeWidth="0.0" text="Notes:" textAlignment="RIGHT" wrappingWidth="120.6640625" GridPane.rowIndex="5">
             <font>
-                <Font name="Calibri" size="24.0"/>
+                <Font name="Calibri" size="24.0" />
             </font>
         </Text>
-        <TextArea fx:id="notesText" prefHeight="200.0" prefWidth="200.0" GridPane.columnIndex="1"
-                  GridPane.columnSpan="4" GridPane.rowIndex="5" GridPane.rowSpan="4">
+        <TextArea fx:id="notesText" prefHeight="200.0" prefWidth="200.0" GridPane.columnIndex="1" GridPane.columnSpan="4" GridPane.rowIndex="5" GridPane.rowSpan="4">
             <GridPane.margin>
-                <Insets bottom="10.0" left="10.0" right="10.0" top="10.0"/>
+                <Insets bottom="10.0" left="10.0" right="10.0" top="10.0" />
             </GridPane.margin>
         </TextArea>
-        <DatePicker fx:id="datePicker" maxHeight="30.0" maxWidth="150.0" prefHeight="30.0" GridPane.columnIndex="1"
-                    GridPane.rowIndex="1">
+        <DatePicker fx:id="datePicker" maxHeight="30.0" maxWidth="150.0" prefHeight="30.0" GridPane.columnIndex="1" GridPane.rowIndex="1">
             <GridPane.margin>
-                <Insets left="10.0"/>
+                <Insets left="10.0" />
             </GridPane.margin>
         </DatePicker>
-        <Text fx:id="clearedLabel" strokeType="OUTSIDE" strokeWidth="0.0" text="Cleared:" wrappingWidth="89.6640625"
-              GridPane.columnIndex="2" GridPane.rowIndex="1">
+        <Text fx:id="clearedLabel" strokeType="OUTSIDE" strokeWidth="0.0" text="Pending?" wrappingWidth="97.6640625" GridPane.columnIndex="2" GridPane.rowIndex="1">
```

Comment:
What does that mean..?

I had a pr to change the controller to accurately reflect our transaction objects and to display the pending field correctly. How are you handling these changes